test/ExpressionSet_basic_test.py

```diff
@@ -67,31 +69,31 @@ def setUpClass(cls):
             os.environ["SDK_CALLBACK_URL"],
             cls.dummy_path,
             "fake_annotation",
-            cls.wsName,
+            WS_NAME,
             cls.wsClient,
         )
 
         # Now we can phony up some expression objects to build sets out of.
         # name, genome_ref, annotation_ref, alignment_ref, ws_name, ws_client
-        cls.expression_refs = list()
+        cls.expression_refs = []
         for idx, alignment_ref in enumerate(cls.alignment_refs):
             cls.expression_refs.append(
                 make_fake_expression(
                     os.environ["SDK_CALLBACK_URL"],
                     cls.dummy_path,
-                    "fake_expression_{}".format(idx),
+                    f"fake_expression_{idx!s}",
```

Comment:
Was the `!s` bit added by Ruff? It's not really necessary, but if autogenerated, it's fine.